WcaOnRails/app/models/competition.rb

```diff
@@ -222,6 +235,7 @@ def update_foreign_keys_when_id_changes
       CompetitionMedium.where(competitionId: id_was).update_all(competitionId: id)
       CompetitionDelegate.where(competition_id: id_was).update_all(competition_id: id)
       CompetitionOrganizer.where(competition_id: id_was).update_all(competition_id: id)
+      DelegateReport.where(competition_id: id_was).update_all(competition_id: id)
```

Comment:
What I was thinking was something like this:

    FKS = { CompetitionMedium => :competitionId, CompetitionDelegate => :competition_id, CompetitionOrganizer => :competition_id, DelegateReport => :competition_id, Registration => :competitionId, Result => :competitionId, Scramble => :competitionId }
    FKS.each{ |model, column| model.(column => id_was).update_all(column => id) }

The test would loop through all Model classes and find those with columns named `competitionId` or `competition_id`.

---
That would be fine, but... let's look at some other ideas.

For one thing, if we can find these columns in a test, why not use that in the real code? Assuming servers restart after each schema migration, you could cache that data, and just keep going. I think I ultimately don't like that, because what gets updated will just silently change without any human eyes verifying it's correct.

---
But what really gets me is that we _have already told AR_ about these columns. It's right there in the `has_many